src/redisearch_api.c

```diff
@@ -368,23 +379,55 @@ QueryNode* RediSearch_CreateGeoNode(IndexSpec* sp, const char* field, double lat
   return ret;
 }
 
-QueryNode* RediSearch_CreatePrefixNode(IndexSpec* sp, const char* fieldName, const char* s) {
+static QueryNode* RediSearch_CreateAffixNode(IndexSpec* sp, const char* fieldName, const char* s,
```

Comment:
`RediSearch_CreateAffixNode` typo?

Reply:
affix, a grammatical element that is combined with a word, stem, or phrase to produce derived or inflected forms. There are three main types of affixes: prefixes, infixes, and suffixes.